recipes/mfast/all/conanfile.py

```diff
@@ -71,10 +71,12 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("boost/1.75.0")
+        # mfast<=1.22.0 doesn't support boost>=1.76.0
+        # see https://github.com/objectcomputing/mFAST/issues/118
```

Comment:
Thanks a lot for the comment! And to @SpaceIm for reporting it upstream in the first place :)